### ISSUE TITLE
Remove uuid from svcat get output

### ISSUE THREAD

**Issue by carolynvs** [kubernetes-incubator/service-catalog#1804]
When using svcat, the uuid loses its importance because svcat understands how to work with the external names. I propose that we not display uuid in the `svcat get` command output. When someone wants to know the uuid, fore example so that they can use it in a kubectl command, they can look it up with `svcat describe`.

In it's place, I'd like to make the broker more visible (see https://github.com/kubernetes-incubator/service-catalog/issues/1749). I'm showing the final output that I'd like even though these are two separate issues (this one is just for not displaying uuid)_. 😀 

```console
$ svcat get classes
            NAME                      BROKER                         DESCRIPTION
+--------------------------+---------------------------+--------------------------------------+
  azure-rediscache            osba                        Azure Redis Cache
                                                          (Experimental)
  redis                       Minibroker                  In-cluster installation of Redis

$ svcat get plans
        NAME                  CLASS                      BROKER                   DESCRIPTION
+------------------+--------------------------+----------------------+----------------------------------------+
  basic              azure-rediscache           osba                    Basic Tier, 250MB Cache
  redis-5-7          redis                      Minibroker              In-cluster installation of Redis 5.7
```

```console

```

**Comment by n3wscott** [comment]
I think there is still some value in being able to query based on UUID, I hope that aspect was not removed. Maybe the uuid column can be optionally shown?

**Comment by carolynvs** [comment]
Nothing has been implemented yet, this is just a suggestion. It wasn't my intention of removing the `--uuid` flag supporting uuid queries. I just want remove it from the default columns displayed in the `get` output table.

**Comment by carolynvs** [comment]
Closed by #1977
